examples/jaeger-hotrod/docker-compose.yml

```diff
@@ -54,7 +54,7 @@ services:
       - my_network
   opensearch:
     container_name: node-0.example.com
-    image: opensearchproject/opensearch:2
+    image: opensearchproject/opensearch:2.9.0
```

Comment:
We should make the OpenSearch versions match. So update this to 2.11.0 or bring Dashboards to 2.9.0.

Reply:
Done! with 2.9.0

It seems like 2.11.0 is not working.